network/pulsenetwork/distributor.go

```diff
@@ -257,45 +251,34 @@ func (d *distributor) sendPulseToHost(ctx context.Context, p *insolar.Pulse, hos
 
 	pulseRequest := NewPulsePacket(ctx, p, d.pulsarHost, host, uint64(d.generateID()))
 
-	call, err := d.sendRequestToHost(ctx, pulseRequest, host)
-	if err != nil {
-		return err
-	}
-	_, err = call.WaitResponse(d.pulseRequestTimeout)
+	_, err := d.sendRequestToHost(ctx, pulseRequest, host)
 	if err != nil {
 		return err
 	}
-
 	return nil
 }
 
 func (d *distributor) pause(ctx context.Context) {
```

Comment:
remove this pause/resume methods and use d.pool.Reset() where needed

Reply:
let's experiment with github ...